bootstrap/README.md

```diff
@@ -77,3 +84,59 @@ cargo run --release --example compute_hashes -- --file ./canister_state.bin > ch
 ```
 
 The hash of each chunk is saved in `chunk_hashes.txt`.
+
+## 7. Hardcode the caches into the uploader canister
```

Comment:
The instructions below here are quite specific for testnets. Can we add a note perhaps that clarifies that the remaining instructions are testnet-specific?